### PR TITLE
Homework ChukhSasha 1.

### PR DESCRIPTION
**Pull request by Salvator8622**

### CODE REVIEW

**Review by RyszardP — APPROVED** [review]
Rewieved

File: english/chukh_alex/l1_t1/hwl1.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+1 He usually has breakfast at 6 a.m.
+2 Anyone who smokes 20 cigarettes a day can expect to have health problems.
+3 She doesn't like cooking, but today she is baking a pie.
+4 I am finishing) the project right now.
+5 I don’t listen to rock music very often.
+6 — What does your sister do?
+— She is a programmer.
+7 My train leaves at 5.
+8 He (never tidies up his room!!! (irritation)
```
Comment (PrinceVIS) [inline english/chukh_alex/l1_t1/hwl1.md:15]: He is never tidying up his room

File: english/chukh_alex/l1_t1/hwl1.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+1.2 Correct the grammar mistakes
+
+1 Do you usually work on Saturdays?
+2 Listen! Somebodys are singing.
```
Comment (PrinceVIS) [inline english/chukh_alex/l1_t1/hwl1.md:20]: sombedy`s singing (words sb, sth don`t have plural forms)

**Review by PrinceVIS — APPROVED** [review]
Found some minor mistakes in Eng HW

File: english/chukh_alex/l1_t1/hwl1.md (hunk 1 above)
Comment (Salvator8622) [reply english/chukh_alex/l1_t1/hwl1.md:15]: oh, realy, this is an action that is permanent, didn't see

File: english/chukh_alex/l1_t1/hwl1.md (hunk 2 above)
Comment (Salvator8622) [reply english/chukh_alex/l1_t1/hwl1.md:20]: ty)

**Review by GrandCkay — APPROVED** [review]

**Pull request closed without merging**